src/app-sandbox/index.tsx

```diff
@@ -9,23 +10,52 @@ export interface Properties {
   web3Provider: any;
   znsRoute: string;
   selectedApp: Apps;
+  hasOverlay: boolean;
+  isOpen: boolean;
 }
 
 export class AppSandbox extends React.Component<Properties> {
   renderSelectedApp() {
     const { znsRoute, selectedApp: app, web3Provider } = this.props;
 
     if (app === Apps.Feed) {
-      return <FeedApp route={{znsRoute, app}} provider={web3Provider} />;
+      return <FeedApp route={{ znsRoute, app }} provider={web3Provider} />;
     }
 
-    return <div className='error'>Error {app} application has not been implemented.</div>
+    return (
+      <div className='error'>
+        Error {app} application has not been implemented.
+      </div>
+    );
+  }
+
+  renderOverlay() {
+    return (
+      <div className='overlay__container'>
+        <div
+          className={classNames(
+            'overlay',
+            { overlay__open: this.props.isOpen },
+            { overlay__closed: !this.props.isOpen }
+          )}
+        >
+          <div className='overlay__content'>
+            <div className='overlay__first'></div>
+            <div className='overlay__secend'></div>
+          </div>
+        </div>
+        )
+      </div>
+    );
   }
 
   render() {
     return (
-      <div className="app-sandbox">
+      <div className='app-sandbox'>
         {this.renderSelectedApp()}
+        {this.props.hasOverlay && 
```

Comment:
Is it possible to use the  `isOpen` var in place of the `hasOverlay` var or vice versa? What scenario could we have an overlay that is not opened?